nebullvm/optimizers/tensor_rt.py

```diff
@@ -246,3 +277,85 @@ def optimize(
             if not is_valid:
                 return None
         return learner
+
+    def optimize_from_torch(
+        self,
+        torch_model: torch.nn.Module,
+        model_params: ModelParams,
+        input_tfms: MultiStageTransformation = None,
+        metric_drop_ths: float = None,
+        quantization_type: QuantizationType = None,
+        metric: Callable = None,
+        input_data: DataManager = None,
+    ) -> Optional[PytorchTensorRTInferenceLearner]:
+        self._log(
+            f"Optimizing with {self.__class__.__name__} and "
+            f"q_type: {quantization_type}."
+        )
+        if not torch.cuda.is_available():
+            raise SystemError(
+                "You are trying to run an optimizer developed for NVidia gpus "
+                "on a machine not connected to any GPU supporting CUDA."
+            )
+
+        supported_q_types = [
```

Comment:
Does TensorRT's PyTorch interface accept only full precision and half precision? ONNX's also supports 8-bit quantization! I would be quite surprised if this is not the case for PyTorch as well.